QualityControl/lib/controllers/LayoutController.js

```diff
@@ -229,33 +215,22 @@ export class LayoutController {
    */
   async patchLayoutHandler(req, res) {
     const { id } = req.params;
-    if (!id) {
-      updateAndSendExpressResponseFromNativeError(res, new InvalidInputError('Missing ID'));
-    } else {
-      let layout = {};
-      try {
-        layout = await LayoutPatchDto.validateAsync(req.body);
-      } catch {
-        updateAndSendExpressResponseFromNativeError(
-          res,
-          new InvalidInputError('Invalid request body to update layout'),
-        );
-        return;
-      }
-
-      try {
-        await this._dataService.readLayout(id);
-      } catch {
-        updateAndSendExpressResponseFromNativeError(res, new NotFoundError(`Unable to find layout with id: ${id}`));
-        return;
-      }
-      try {
-        const layoutUpdated = await this._dataService.updateLayout(id, layout);
-        res.status(201).json(layoutUpdated);
-      } catch {
-        updateAndSendExpressResponseFromNativeError(res, new Error(`Unable to update layout with id: ${id}`));
-        return;
-      }
+    let layout = {};
+    try {
+      layout = await LayoutPatchDto.validateAsync(req.body);
+    } catch {
+      updateAndSendExpressResponseFromNativeError(
+        res,
+        new InvalidInputError('Invalid request body to update layout'),
+      );
+      return;
+    }
+    try {
+      const layoutUpdated = await this._dataService.updateLayout(id, layout);
```

Comment:
Similar to the other method, might be good to be more explicit with the naming:
```suggestion
      const layoudId = await this._dataService.updateLayout(id, layout);
```

Reply:
Done